templates/CRM/Event/Form/Participant.tpl

```diff
@@ -28,7 +28,7 @@
     <div class="view-content">
       {if !empty($participantMode)}
         <div class="help">
-          {ts 1=$displayName 2=$participantMode|upper}Use this form to submit an event registration on behalf of %1. <strong>A %2 transaction will be submitted</strong> using the selected payment processor.{/ts}
+          {ts 1=$displayName 2=$participantMode|crmUpper}Use this form to submit an event registration on behalf of %1. <strong>A %2 transaction will be submitted</strong> using the selected payment processor.{/ts}
```

Comment:
This is out of scope but this seems to be an internal variable serving double-duty as a front-end label. At one point it's even compared to the literal `test`. Something to look at on a rainy day.